testing/mock/test_system.h

```diff
@@ -30,7 +30,7 @@
 #include <map>
 #include <string>
 #include <vector>
-
+#include "opae/fpga.h"
```

Comment:
Not needed here

Reply:
I could be wrong but I need to include that to define fpga_version and fpga_accelerator type inside test_device struct. There is probably other ways to do this.

Reply:
Sorry, it should be <opae/fpga.h>